kube-burner-workload/kind-metrics/README.md

```diff
@@ -0,0 +1,60 @@
+This folder helps you enable metric collection for scale tests.
+It consists of the following steps:
+1. Install Prometheus in a KinD cluster
+2. Run Elasticsearch locally with docker
+3. Run Grafana locally with docker
+4. Run kube-burner with metrics collection
+5. Configure Grafana dashboard to collect data from opensearch
```

Comment:
It would be more clear to use either opensearch or elasticsearch in all cases. Given that you added Elasticsearch, we should replace all opensearch (3 occasions in this file) with elasticsearch.

Reply:
good catch! I tried to remove opensearch completely, but obviously that didn't work :)